alg/teca_dataset_diff.h

```diff
@@ -41,18 +41,30 @@ class teca_dataset_diff : public teca_algorithm
     TECA_GET_ALGORITHM_PROPERTIES_DESCRIPTION()
     TECA_SET_ALGORITHM_PROPERTIES()
 
-    // Relative tolerance below which two floating-point quantities are
-    // considered equal (1e-6). The relative difference for a computed
-    // quantity A and a reference quantity B is
-    //
-    // rel_diff = |A - B| / B, B != 0
-    //          = |A - B| / A, B == 0, A != 0
-    //            0            otherwise
+    /** @anchor tolerance
+     * @name tolerance
+     * Set tolerance. Relative tolerance below which two floating-point
```

Comment:
Set the relative tolerance ...

Note: this class has changed substantially and you might be  better off copying the things you want to keep than trying to resolve conflicts